website/guide/getting-started.md

```diff
@@ -157,3 +153,15 @@ If you are using GitHub Actions, you need to setup the NPM token to release pack
 4. Click "Add a new secret".
    - Name: `NPM_AUTH_TOKEN`
    - Value: Paste the token from clipboard.
+
+## Setup with 'Nothing'
```

Comment:
```suggestion
## Setup for manual release
```

Reply:
Are you saying that "set up for manual release" is more appropriate than "set up with nothing"?

Reply:
Yes, I think "set up with nothing" isn't totally clear, as your use case is when you don't have CI and release from your computer. I think the guide can reflect that.

Reply:
I completely agree with you. How about adding a little more content? I think this is more explicit. 'set up for manual release(when 'Nothing' is selected in CI configure)'. Because as you know, the first time you run the 'shipjs setup' command, you can choose between 'Circle CI', 'Github Actions', and 'Nothing'.

Reply:
It was pleasure to contribute! I wonder how do I add in contributor list. Do I run the command in this url? - https://community.algolia.com/shipjs/guide/contributing.html#working-on-code

Reply:
yes, go ahead @Kimjipang, hope to see more of your contributions :)